app/components/autocomplete/at_mention_group/at_mention_group.tsx

```diff
@@ -66,8 +76,8 @@ const GroupMentionItem = ({
         [insets.left, insets.right, style],
     );
     const completeMention = useCallback(() => {
-        onPress(completeHandle);
-    }, [onPress, completeHandle]);
+        onPress(displayName);
```

Comment:
Just confirming, do we want to autocomplete with the display name? Maybe I am missing something, but shouldn't we autocomplete with the name?

Reply:
this definitely needs to use the `name` instead of the `displayName`

Reply:
Whoops - I think I was testing something in there, will change.